modules/quickstart-public-cloud/pages/setup.adoc

```diff
@@ -1,54 +1,39 @@
 [[quickstart-publiccloud-setup]]
-= Setting Up
+= Setting up
 
-We have tested these instructions on Amazon EC2, Google Compute Engine, and Microsoft Azure, but they should work on other providers as well, with some variation.
+This guide shows you the fastest way to get {productname} up and running in a public cloud using on-demand or BYOS services.
+We have tested using {productname} on Amazon EC2, Google Compute Engine, and Microsoft Azure, but they should work on other providers as well, with some variation.
```

Comment:
What do you mean with "they" here?

Reply:
heh, the sentence used to have "these instructions" in it. How about this:

```suggestion
We have tested using {productname} on Amazon EC2, Google Compute Engine, and Microsoft Azure, but these procedures should work with other public cloud providers as well, with some variation.
```

Reply:
LGTM, thanks! :)